helios/media/static_templates/question.html

```diff
@@ -1,10 +1,20 @@
-
 {#foreach $T.questions as question}
 <div id="q_view_{$T.question$index}">
+<h4><strong>{$T.question$index + 1}&ordf; Question</strong>
```

Comment:
Also, to complete my answer... i did this because in our use for helios some users register some big questions and it kind mess up the interface.
Also, perhaps could be a good approuch in limit the number of chars typed both in question and in answers...

Question to 2048 chars and answer some kind of 1024 chars?